packages/react/src/components/Footer/README.md

```diff
@@ -49,7 +49,14 @@ FOOTER_LOCALE_BUTTON=true
 > [.env.example](https://github.com/carbon-design-system/ibm-dotcom-library/blob/master/packages/react/.env.example)
 > for more information
 
-## Types (optional)
+## Props
+
+| Name         | Required | Data Type | Default Value | Description                        |
+| ------------ | -------- | --------- | ------------- | ---------------------------------- |
+| `navgiation` | NO       | Object    | null          | Navigation data object for Footer  |
```

Comment:
@annawen1 typo!

Reply:
ok fixed, updated README's for some incoming components I missed